### ISSUE TITLE
File not showing up in file directory when kindle is connected to pc - unless renamed

### ISSUE THREAD

**Issue by 302scouting** [SUPERCILEX/Robot-Scouter#189]
When we connect our Kindles to a PC, the exported excel document is not showing up in the folder in which it is located. (It shows up in the Kindle File Directory accessed from the Kindle, but is not in the directory when the kindle is connected to a PC and the same folder is accessed.) Here is an example exported file name as `[UNKNOWN_TEMPLATE] 33 - Killer Bees and 302 - The Dragons`. When I rename the file by removing the `[UNKNOWN_TEMPLATE]` part, it shows up. Is there a way to stop the file from having the `[UNKNOWN_TEMPLATE]` part of the name?

**Comment by SUPERCILEX** [comment]
And you're using Windows? I'm able to see `[UNKNOWN_TEMPLATE] 1 - The Juggernauts and 2 - Team 2.xlsx` from Windows. The prefix is supposed to tell you what template the file contains, but I'm guessing you deleted it.

**Comment by 302scouting** [comment]
Not sure exactly what you are asking if we deleted (nothing that I can
think of), but the prefix we've always seen is "Unknown_Template." I can
upload screenshots if you would like.

On Mar 3, 2018 4:45 PM, "Alex Saveau" <notifications@github.com> wrote:

And you're using Windows? I'm able to see [UNKNOWN_TEMPLATE] 1 - The
Juggernauts and 2 - Team 2.xlsx from Windows. The prefix is supposed to
tell you what template the file contains, but I'm guessing you deleted it.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/SUPERCILEX/Robot-Scouter/issues/189#issuecomment-370182291>,
or mute the thread

**Comment by SUPERCILEX** [comment]
Ohhh, I see what's going on: you didn't name your templates (you can do so by long-pressing the template's tab). I've added a fix so that templates without names don't add the prefix, but you can still name your templates if you want and get the correct prefix.

TL;DR: if you just update Robot Scouter and do nothing else, the prefix will be gone. 👍

**Comment by 302scouting** [comment]
Thank you, Alex! We appreciate all you're doing to make scouting successful.
